DevTools/src/TokamakDevTools/Sources/TokamakDevTools/ContentView.swift

```diff
@@ -0,0 +1,31 @@
+import TokamakDOM
+
+struct ContentView: View {
+  @EnvironmentObject var tree: Tree
+  @State private var onlyComposite: Bool = true
+  @State private var query: String = ""
+
+  @ObservedObject var inspector = NodeInspector()
+
+  var body: some View {
+    ScrollView {
+      HStack {
+        Toggle("Only Composite Views", isOn: $onlyComposite)
+        TextField("Search", text: $query)
+      }
+      HStack {
+        // OutlineGroup(tree.root, children: onlyComposite ? \.compositeChildren : \.children) {
+        //     Text(String($0.type.split(separator: "<").first!))
+        //         .foregroundColor($0.isPrimitive ? .primary : ($0.isHost ? .green : .blue))
+        //         .font(.system(size: 12, weight: $0.isPrimitive ? .regular : .bold, design: .monospaced))
```

Comment:
<!--
  1 failure:  Line should be 10...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: DevTools/src/TokamakDevTools/Sources/TokamakDevTools/ContentView.swift;
  Line: 20;
-->
- :no_entry_sign: Line should be 100 characters or less: currently 107 characters (`line_length`)